temoa/data_io/hybrid_loader.py

```
@@ -629,6 +635,101 @@ def _load_existing_capacity(
+            'SELECT region, period, tech, vintage, cap_early FROM output_retired_capacity WHERE '
+            'period <= ? AND scenario = ? AND cap_early > 0 ',
+            (prev_period, self.config.scenario),
+        ).fetchall()
+
+        self._load_component_data(data, model.retired_existing_capacity, rows_to_load)
+
+    # --- Lifetime components ---
+    def _load_lifetime_tech(
+        self,
+        data: dict[str, object],
+        raw_data: Sequence[tuple[object, ...]],
```

Ruff (0.15.18)

[warning] Unused method argument: `raw_data`

(ARG002)